Makefile

```
@@ -1,5 +1,11 @@
 VERSION := v0.1.0
 
+ORG_PATH := github.com/Azure
+PROJECT_NAME := eraser
+REPO_PATH := $(ORG_PATH)/$(PROJECT_NAME)
+BUILD_COMMIT := $(shell git rev-parse --short HEAD)
+BUILD_TIMESTAMP := $$(date +%Y-%m-%d-%H:%M)
```

```suggestion
SOURCE_DATE_EPOCH ?= $$(shell date -u --date=$(git show -s --format=%cI HEAD) +%s)
```

`SOURCE_DATE_EPOCH` is a more standard var here.
Using the commit time as the timestamp makes the build more reproducible.
This also lets the builder inject their own timestamp.